src/SynthImage/line_extraction.py

```diff
@@ -52,7 +52,7 @@ def get_max_width(self, lines, blank_img, font):
             # Draw the line on the blank image to calculate the bounding box
             temp_img = blank_img.copy()
             temp_draw = ImageDraw.Draw(temp_img)
-            line_bbox = temp_draw.textbbox((20, y), line, font=font)
+            line_bbox = temp_draw.textbbox((10, y), line, font=font)
```

Comment:
Hard coded constant is not good practice. Please initialise a constant and use it. The best practice would be creating a config.py in the pkg and intialise the constant over there and import it to respective module where need.

Reply:
ok sure, i will do that @kaldan007